gulpfile.js

```
@@ -60,24 +60,45 @@ const pluginGlobs = [
     repo + 'org.osgi.service.repository_**',
+    repo + 'org.eclipse.m2e.versions.model_**',
+    repo + 'org.eclipse.m2e.versions.common_**',
+    repo + 'org.osgi.service.event_**',
+    repo + 'org.apache.commons.commons-collections4_**',
+    repo + 'stax2-api_**',
+    repo + 'org.eclipse.m2e.versions.api_**',
     repo + 'org.apache.commons.commons-io_**'
 ];
 
 gulp.task('patch_version', (cb) => {
     const packageJsonData = require('./package.json');
```

ESLint

[error] A `require()` style import is forbidden.

(`@typescript-eslint/no-require-imports`)

---

[error] 'require' is not defined.

(no-undef)